addons/l10n_fi/data/account_tax_report_line.xml

```diff
@@ -13,7 +13,7 @@
     <record id="tax_report_sales_25_5" model="account.tax.report.line">
         <field name="name">25.5 %:n vero</field>
```

Comment:
Spoke to @Olivier-olma, we should include a description something like "(includes 24%)"

Reply:
EDIT:
or "(+ 24 %:n vero)" 
or rather, maybe we should add the comment on the 24% line - that way we won't have to change the 25.5% line in the future?
I leave it up to you

Reply:
I'd rather not tell what's in the 25.5 line in the 24.0 one so I put "(+ 24 %:n vero)" in the 25.5% line name. Anyway we will have multiple changes to apply on the first of january 2025 (i guess) so changing the line's name in the report is no big deal. Thnak you for asking Olma :+1: